omero/sysadmins/version-requirements.rst

```diff
@@ -103,53 +81,48 @@ The following subsections detail the versions of each operating system
 which are supported by both its upstream developers (for security and
 general updates) and by OME for OMERO building and server deployment.
 
-UNIX (FreeBSD)
---------------
-
-It only really makes sense to support the base toolchain for major
-releases and the Ports tree (which is continually updated); these will
-be covered in the dependencies, below.
 
 Linux (CentOS and RHEL)
 -----------------------
 
 General overview for `RHEL
 <https://access.redhat.com/articles/3078>`__ and `CentOS
-<https://wiki.centos.org/About/Product>`__
+<https://www.centos.org/>`__.
 
 .. list-table::
     :header-rows: 1
 
     * - Version
       - Release date
       - Supported until
-      - OMERO 5.4
-      - OMERO 5.5
       - OMERO 5.6
-      - Details
-    * - 6
-      - Nov 2010
-      - Nov 2020
-      - |Deprecated|
-      - |Dropped|
-      - |Dropped|
-      - `Reference <https://wiki.centos.org/About/Product>`__
-    * - 7
+    * - `RHEL 7 <https://endoflife.date/rhel>`__
       - June 2014
       - June 2024
+      - |Supported|
```

Comment:
I think `|Deprecated|` would be inline with https://www.openmicroscopy.org/2023/07/24/linux-distributions.html